pkg/k8shandler/configmap.go

```diff
@@ -24,6 +24,22 @@ func NewConfigMap(configmapName string, namespace string, data map[string]string
 	}
 }
 
+//NewConfigMap stubs an instance of Configmap with labels
+func NewConfigMapWithLabels(configmapName string, namespace string, labels map[string]string, data map[string]string) *core.ConfigMap {
+	return &core.ConfigMap{
```

Comment:
instead can we leverage the `NewConfigMap` function and then just add labels to the object metadata?